### ISSUE TITLE
Publishing the IPFS pacman.store on Arch Wiki

### ISSUE THREAD

**Issue by GreenLunar** [RubenKelevra/pacman.store#64]
After reading the post of [HalosGhost](https://bbs.archlinux.org/profile.php?id=61234) at [2021-07-27 15:51:34](https://bbs.archlinux.org/viewtopic.php?pid=1984957#p1984957), I don't know whether I may or may not post a continuation to that thread, so I'm posting here.

**Arch Wiki**
Publishing the IPFS pacman.store mirror on Arch Wiki and official mirror list.

---

I also wanted to post an article for the sake of productivity and open a discussion on the forum... so I'm posting a link to it here too http://www.pragma-grid.net/images/pragma34/Tipchuen.pdf

**Comment by RubenKelevra** [comment]
Yeah, not sure why an old thread cannot be updates with new informations.

Would be cool to have more documentation. Especially since there's an extention for pacman on the way to have cache servers:

https://github.com/RubenKelevra/pacman.store/issues/41#issuecomment-1019225915

**Comment by RubenKelevra** [comment]
@GreenLunar wrote:

> I also wanted to post an article for the sake of productivity and open a discussion on the forum... so I'm posting a link to it here too http://www.pragma-grid.net/images/pragma34/Tipchuen.pdf


Not sure how relevant this graph is: 😆 

```
The APT-IPFS performance results reported in the graph show significant performance
gains over APT when a user use APT-IPFS to download software that have been
downloaded before.
```

That's basically what pacman already does: Local caching. Important thing is regional internet caching and local network caching. :)
